File: src/frontend/src/app/components/financeiro/operacao-caixa/model/caixa-com-status-dto.ts
import { StatusAberturaCaixa } from './status-abertura-caixa.enum';

export interface CaixaComStatusDTO {
  caixaId: string;
  caixaNome: string;
  valorPadraoAbertura: number;
  statusSessao: StatusAberturaCaixa | null;
  aberturaCaixaId: string | null;
  dataAbertura: string | null;
  usuarioNomeAbertura: string | null;
}

File: src/backend/src/main/java/br/com/grupopipa/gestaointegrada/financeiro/aberturacaixa/CaixaComStatusDTO.java
package br.com.grupopipa.gestaointegrada.financeiro.aberturacaixa;

import java.math.BigDecimal;
import java.time.LocalDateTime;
import java.util.UUID;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class CaixaComStatusDTO {
    private UUID caixaId;
    private String caixaNome;
    private BigDecimal valorPadraoAbertura;
    private StatusAberturaCaixa statusSessao;
    private UUID aberturaCaixaId;
    private LocalDateTime dataAbertura;
    private String usuarioNomeAbertura;
}
